package.json

```diff
@@ -95,6 +95,12 @@
     "update-changelog": "auto-changelog --commit-limit false --package",
     "update-license": "node scripts/generate-license.js && git add LICENSE"
   },
+  "directories": {
+    "test": "test"
```

Comment:
any reason?

Reply:
just alphabetical order. for what tool is that useful bytheway?

Reply:
oh, I didn't realize this was here in the first place.

looks like it was added for grunt back in the day. it did what [you did here](https://github.com/Modernizr/Modernizr/blob/master/gulpfile.babel.js#L16-L30)

id say chuck it, while we are here

Reply:
chucked it!

Reply:
![image](https://media.giphy.com/media/zqhZB6bo5FgoE/giphy.gif)